packages/core/src/components/modal/modal.tsx

```diff
@@ -86,6 +87,13 @@ export class Modal {
    */
   @Event() dialogDismiss: EventEmitter;
 
+  @OnListener<Modal>('keydown', (self) => !self.keyboard)
```

Comment:
Why is global event listener here necessary?

Reply:
Because the onKey listener will only get called when the dialog has focus. Clicking on it removes the focus.